crates/compiler/test_mono/generated/if_multi_branch.txt

```diff
@@ -1,13 +1,21 @@
+procedure Bool.1 ():
+    let Bool.11 : Int1 = false;
+    ret Bool.11;
+
+procedure Bool.2 ():
+    let Bool.12 : Int1 = true;
+    ret Bool.12;
```

Comment:
This is **begging** for inlining ;)

A note aside: we might want to special case this in mono so that programs that don't actually use anything in `Bool` other than true/false don't have to enter into specializing `Bool` (which is a marginal cost, but a cost nonetheless, because a thread has to take the time to generate these two functions, but we know we can unconditionally inline them). That doesn't need to be done in this PR, though.